README.md

```diff
@@ -4,7 +4,7 @@ React is a JavaScript library for building user interfaces.
 
 * **Declarative:** React makes it painless to create interactive UIs. Design simple views for each state in your application, and React will efficiently update and render just the right components when your data changes. Declarative views make your code more predictable, simpler to understand, and easier to debug.
 * **Component-Based:** Build encapsulated components that manage their own state, then compose them to make complex UIs. Since component logic is written in JavaScript instead of templates, you can easily pass rich data through your app and keep state out of the DOM.
-* **Learn Once, Write Anywhere:** We don't make assumptions about the rest of your technology stack, so you can develop new features in React without rewriting existing code. React can also render on the server using Node and power mobile apps using [React Native](https://facebook.github.io/react-native/).
+* **Learn Once, Write Anywhere:** We don't make assumptions about the rest of your technology stack, so you can develop new features in React without rewriting the existing code. React can also render on the server using Node and power mobile apps using [React Native](https://facebook.github.io/react-native/).
```

Comment:
The original seems grammatically correct to me. In fact, I think that this change introduces some ambiguity. Which existing code do we not have to rewrite? By adding "the", "existing code" could refer to either the React codebase or your application codebase.

If the sentence needs clarification, something like the following could work, but I'm not convinced that it adds anything to the sentence since I think it's implied by "so you".

> We don't make assumptions about the rest of your technology stack, so you can develop new features in React without rewriting **_your_** existing code.